mmaction/__init__.py

```diff
@@ -4,7 +4,7 @@
 from .version import __version__
 
 mmcv_minimum_version = '1.1.1'
-mmcv_maximum_version = '1.2.0'
+mmcv_maximum_version = '1.3.0'
 mmcv_version = digit_version(mmcv.__version__)
 
 assert (digit_version(mmcv_minimum_version) <= mmcv_version
```

Comment:
<

Reply:
<= is okay

Reply:
I think `<=` is more reasonable, isn't it ?

Reply:
Minor version is breaking, isn't it?

Reply:
`<= 1.3` means any version of 1.3 is excluded, such as 1.3.0, 1.3rc0, etc.

Reply:
If you allow 1.3, why not simply bound on v2?

Reply:
1.3.0 is not allowed.

Reply:
`digit_version('1.3rc0') < digit_version('1.3.0')` will return True, but we assume that 1.3rc0 also breaks the compatibility.

Reply:
So lets bound on 1.3rc0?

Reply:
okay 1.3